app/main/views/agreements.py

```diff
@@ -62,28 +62,43 @@ def list_agreements(framework_slug):
 @login_required
 @role_required('admin', 'admin-ccs-sourcing')
 def next_agreement(supplier_id, framework_slug):
+    status = request.args.get("status")
```

Comment:
This is a good question. I have no idea as to the answer. I think alexey's reaction that there was a distinction between `approved` and `countersigned` in the backend at all was surprise, so perhaps it's an unnecessary distinction...?